style(home): avoid horizontal scroll on mobile devices

fidele137 opened the pull request:
The content overflows horizontally on mobile devices due to fixed width of images. This is fixed using a max-width. Also the font-size tends to be too big so it is reduced for small viewports

manekinekko commented:
Thank you @fidele137 for your PR. Much appreciated :)

manekinekko commented:
Hi @fidele137 can I ask you to push an extra empty commit (--amend) on this branch? I'd like to validate some extra build checks we just added to this repo. Your next commit should trigger these new checks. Cheers ;)

fidele137 commented:
> Hi @fidele137 can I ask you to push an extra empty commit (--amend) on this branch? I'd like to validate some extra build checks we just added to this repo. Your next commit should trigger these new checks. Cheers ;)

I added an empty commit -a as requested and rebased onto the upstream/develop branch.
Thanks :) !

manekinekko commented:
Puuurrfect 👍
